src/main/java/app/linguistai/bmvp/controller/gamification/StoreController.java

```diff
@@ -91,4 +104,18 @@ public ResponseEntity<Object> purchaseUserItem(@RequestHeader(Header.USER_EMAIL)
     public ResponseEntity<Object> decreaseUserItemQuantity(@RequestHeader(Header.USER_EMAIL) String email, @RequestParam @NotBlank UUID itemId) throws Exception {
```

Comment:
Instead of trying to keep item ids up-to-date in the frontend (mobile), wouldn't using enums be better? IDs are not persistant through multiple setups and in local development.

Reply:
I deleted this endpoint and added a check for item's quantity bounds